tests/unit/TechRecordsService.unitTest.ts

```diff
@@ -551,7 +553,7 @@ describe("updateTechRecord", () => {
         };
       });
       const techRecordsService = new TechRecordsService(new MockDAO());
-      const updatedTechRec: any = await techRecordsService.updateTechRecord(techRecord, msUserDetails);
+      const updatedTechRec: any = await techRecordsService.updateTechRecord(techRecord, msUserDetails, undefined);
```

Comment:
maybe oldStatusCode should default to undefined if it's not passed so you don't have to manually update all these function calls